Specification of configure CFLAGS breaks build

When CFLAGS are set during configuration:

```
 ./configure CFLAGS="-ggdb3 -O0"
```

Then, build fails:

```
...
gcc -DHAVE_CONFIG_H -I. -I../../src  -I../../src -I../../src/lib  -DLOCALEDIR=\"/usr/local/share/locale\"   -ggdb3 -O0 -Wall  -DDATADIR="\"/usr/local/share/ddccontrol-db\"" -DBINDIR="\"/usr/local/bin\"" -DPKGLIBEXECDIR="\"\"" -MT sis.o -MD -MP -MF $depbase.Tpo -c -o sis.o sis.c &&\
mv -f $depbase.Tpo $depbase.Po
/bin/bash ../../libtool  --tag=CC   --mode=link gcc  -ggdb3 -O0 -Wall  -DDATADIR="\"/usr/local/share/ddccontrol-db\"" -DBINDIR="\"/usr/local/bin\"" -DPKGLIBEXECDIR="\"\"" -Wl,-z,now  -o ddcpci main.o nvidia.o radeon.o i2c-algo-bit.o intel810.o via.o sis.o  -lpci -lm 
libtool: link: gcc -ggdb3 -O0 -Wall -DDATADIR=\"/usr/local/share/ddccontrol-db\" -DBINDIR=\"/usr/local/bin\" -DPKGLIBEXECDIR=\"\" -Wl,-z -Wl,now -o ddcpci main.o nvidia.o radeon.o i2c-algo-bit.o intel810.o via.o sis.o  -lpci -lm
i2c-algo-bit.o: In function `sdalo':
/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:81: undefined reference to `ssleep'
i2c-algo-bit.o: In function `sdahi':
/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:87: undefined reference to `ssleep'
i2c-algo-bit.o: In function `scllo':
/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:93: undefined reference to `ssleep'
i2c-algo-bit.o: In function `sclhi':
/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:106: undefined reference to `ssleep'
/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:125: undefined reference to `ssleep'
i2c-algo-bit.o:/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:145: more undefined references to `ssleep' follow
i2c-algo-bit.o: In function `bit_xfer':
/home/miroslav/projects/ddccontrol/src/ddcpci/i2c-algo-bit.c:507: undefined reference to `readbytes'
collect2: error: ld returned 1 exit status
Makefile:445: recipe for target 'ddcpci' failed
make[3]: *** [ddcpci] Error 1
make[3]: Leaving directory '/home/miroslav/projects/ddccontrol/src/ddcpci'
Makefile:425: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/home/miroslav/projects/ddccontrol/src'
Makefile:364: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/home/miroslav/projects/ddccontrol/src'
Makefile:441: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,8 +13,14 @@ addons:
       - pciutils-dev
       - libgtk2.0-dev
 
+env:
+  matrix:
+    -
+    - CFLAGS='CFLAGS=-O0 -g'
+    - CFLAGS='CFLAGS=-O2'
+
 script:
   - ./autogen.sh
-  - ./configure
+  - ./configure "${CFLAGS}"
   - make -j4
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - show warning message without blinking
 - install binary ddcpci to pkglibexec directory (it's not run by users)
+- fixed build with custom CFLAGS
 
 ## 0.4.3 - 2017-12-28
 ### Removed

--- a/src/ddcpci/i2c-algo-bit.c
+++ b/src/ddcpci/i2c-algo-bit.c
@@ -18,7 +18,7 @@
     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.		     */
 /* ------------------------------------------------------------------------- */
 
-/* With some changes from Frodo Looijaard <frodol@dds.nl>, Kyösti Mälkki
+/* With some changes from Frodo Looijaard <frodol@dds.nl>, Kyï¿½sti Mï¿½lkki
    <kmalkki@cc.hut.fi> and Jean Delvare <khali@linux-fr.org> */
 
 /* Adapted to user space by Nicolas Boichat <nicolas@boichat.ch> */
@@ -58,7 +58,7 @@ static struct timeval newtv;
 
 /* ----- sleep function ---------------------------------------------	*/
 
-void inline ssleep(const int usec) {
+static inline void ssleep(const int usec) {
    gettimeofday(&lasttv, NULL);
    while (1) {
       gettimeofday(&newtv, NULL);
@@ -378,7 +378,7 @@ int sendbytes(struct i2c_algo_bit_data *adap, struct i2c_msg *msg)
 	return wrcount;
 }
 
-inline int readbytes(struct i2c_algo_bit_data *adap, struct i2c_msg *msg)
+static inline int readbytes(struct i2c_algo_bit_data *adap, struct i2c_msg *msg)
 {
 	int inval;
 	int rdcount=0;   	/* counts bytes read */

